examples/Karma/view.en.shtml

```diff
@@ -107,23 +107,18 @@
+						<br />
```

Comment:
@ChintanRathod this should go in the start.en.shtml page not the view. I will merge this and change it by hand.